### PR TITLE
localstack added to local development enviroment

### PR DESCRIPTION
**Pull request by szdrasiak**

### CODE REVIEW

File: services/workers/serverless.yml [hunk 1]
```diff
@@ -4,41 +4,25 @@ provider:
...
-      Action:
-        - "secretsmanager:GetRandomPassword"
-        - "secretsmanager:CreateSecret"
-        - "secretsmanager:ListSecrets"
-      Resource: '*'
-    - Effect: 'Allow'
-      Action:
-        - "secretsmanager:*"
-      Resource: { Fn::ImportValue: '${self:custom.projectEnvName}-databaseSecretArn' }
+  stackName: ${file(./workers.conf.${self:provider.stage}.yml):stackName}
```
Comment (pziemkowski) [inline services/workers/serverless.yml:9]: I like the file approach, but we only need two files. One for all deployed envs, one for local

File: services/workers/workers.conf.dev.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
+stackName: "${env:PROJECT_NAME}-workers-${self:provider.stage}"
```
Comment (pziemkowski) [inline services/workers/workers.conf.dev.yml:1]: I didn't know there was an option to modify stack name! this is good news.
Can you make this one follow the same pattern as other stacks do?
`${env:PROJECT_NAME}-${env:ENV_STAGE}-workers` ?

Be sure to update CodeBuild roles in `ciServerless.ts`!

File: docker-compose.yml [hunk 3]
```diff
@@ -11,6 +11,26 @@ services:
       - POSTGRES_PASSWORD=backend
       - POSTGRES_DB=backend
 
+  localstack:
+    image: localstack/localstack:0.11.3
+    ports:
+      - "4566:4566"
+    environment:
+      - SERVICES=serverless,events,cloudformation,ses,secretsmanager
+      - DATA_DIR=${DATA_DIR- }
```
[the remark this answers is not in the document]
Comment (szdrasiak) [reply docker-compose.yml:20]: yep is valid, but there is one question, do we want keeping localstack data(s3,dynamo) after we kill app containers? if yes I have to create external volume in docker-compose, but It makes `docker-compose up` slower, depending how much files localstack volume contains

[the remark this answers is not in the document]
Comment (pziemkowski) [reply docker-compose.yml:20]: I think that dynamodb data would need to be persistent.

File: services/workers/serverless.yml [hunk 4]
```diff
@@ -49,11 +33,31 @@ functions:
...
+
+  localstack:
+    stages:
+      - local
+    host: http://localhost
+    endpointFile: localstack_endpoints.json
+    debug: true
+    mountCode: true
+
+  confFile:
```
Comment (pziemkowski) [inline services/workers/serverless.yml:54]: Nice idea!

**Pull request merged**